test/unit/models/forms/FormConstraintsTest.scala

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.forms
+
+import java.util.Calendar
+
+import play.api.data.validation.{Invalid, Valid}
+import uk.gov.hmrc.play.test.UnitSpec
+
+class FormConstraintsTest extends UnitSpec {
+
+  private val currentYear: Int = Calendar.getInstance().get(Calendar.YEAR)
+  //TODO read from appconfig
+  private val yearLowBound: Int = 2010
+
+  "FormConstraintsTest" when {
+
+    "dateMustBeInThePast" should {
+      "return invalid when date entered is tomorrow" in {
+        val date = Calendar.getInstance()
+        date.add(Calendar.DAY_OF_MONTH, 1)
+        val dateInstant = date.toInstant
+        val result = FormConstraints.dateMustBeInThePast("Test error").apply(dateInstant)
+
+        result shouldBe Invalid("Test error")
+      }
+
+      "return invalid when date entered is one month from now" in {
+        val date = Calendar.getInstance()
+        date.add(Calendar.MONTH, 1)
+        val dateInstant = date.toInstant
+        val result = FormConstraints.dateMustBeInThePast("Test error").apply(dateInstant)
+
+        result shouldBe Invalid("Test error")
+      }
+
+      "return valid when date entered is in the past (one month ago)" in {
+        val date = Calendar.getInstance()
+        date.add(Calendar.MONTH, -1)
+        val dateInstant = date.toInstant
+        val result = FormConstraints.dateMustBeInThePast("Test error").apply(dateInstant)
+
+        result shouldBe Valid
+      }
+
+      "return valid when date entered is in the past (1000 years ago)" in {
+        val date = Calendar.getInstance()
+        date.add(Calendar.YEAR, -1000)
+        val dateInstant = date.toInstant
+        val result = FormConstraints.dateMustBeInThePast("Test error").apply(dateInstant)
+
+        result shouldBe Valid
+      }
+    }
+
+    //TODO Pass the correct error message
```

Comment:
Same question here

Reply:
It was fixed as part of last commit, will remove this todo now